### DISCUSSION TITLE
Qual a diferença de utilizar o EF.Tools e o EF.Design?

### DISCUSSION THREAD

**Discussion by paulosfjunior** [balta-io/forum#207] [Tira Dúvidas] [answered]
Gostaria de saber a diferença entre os packages Microsoft.EntityFrameworkCore.Tools e Microsoft.EntityFrameworkCore.Design, li um pouco a respeito mas não consegui clarear a mente.

Obrigado

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @paulosfjunior, como vai?

O `Microsoft.EntityFrameworkCore.Tools` contém recursos para gerar migrations e outras coisas ferramentais. O Design contém definições como tipo de dados no SQL Server, forma de criar índices e outros itens de mapeamento por exemplo.
